pkgs/development/python-modules/google-crc32c/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, pkgs, buildPythonPackage, fetchPypi, cffi }:
+
+buildPythonPackage rec {
+  pname = "google-crc32c";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "lDm5YLbs2EdVdnXRMPw2Jtdiv1NdpZXCCmlJpwX7Pq4=";
+  };
+
+  buildInputs = [ pkgs.crc32c  ];
+  propagatedBuildInputs = [ cffi ];
+
+  preConfigure = ''
+    export LDFLAGS="-L${pkgs.crc32c}/lib"
+    export CFLAGS="-I${pkgs.crc32c}/include"
+  '';
```

attrs passed to mkDerivation should be present as env vars at build time.
```suggestion
    LDFLAGS="-L${crc32c}/lib"
    CFLAGS="-I${crc32c}/include"
  '';
```